tests/integration/conftest.py

```diff
@@ -91,14 +92,28 @@ def addresses() -> List[Wallet]:
     """
     random.seed(0)
     private_keys = [generate_random_private_key() for _ in range(4)]
-    return [
-        Wallet(
-            address=private_key.public_key.to_checksum_address(),
-            private_key=private_key,
-            starknet_address=int(private_key.public_key.to_address(), 16),
+    wallets = []
+    for private_key in private_keys:
+        tempAccount: Account = web3.eth.Account().from_key(private_key)
+        evm_address = web3.Web3().toInt(hexstr=tempAccount.address)
```

Comment:
you could just have used `evm_address = private_key.public_key.to_checksum_address()`